packages/job-dispatch/src/job-variables-deployment/job-variables-deployment.ts

```
@@ -0,0 +1,149 @@
+      value: firstMatchedValue,
+      directMatch: true,
+    };
+
+  if (defaultValue != null)
+    return {
+      value: defaultValue,
+      directMatch: true,
+    };
+
+  return {
+    value: deploymentVariableValues[0]!,
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)